CHANGES.md

```diff
@@ -5,6 +5,7 @@
 - Add "searchBarConfig.showSearchInCatalog" to configParameters so that the link in location search results can be disabled.
 - Properly initialize react ref in functional components
 - Add NominatimSearchProvider.
+- Restrict mobx version to '< 6.13.0' to avoid tsc errors.
```

Comment:
Might be good to add that the change is required because mobx now uses Iterator helper types introduced in typescript version 5.6.x.

Reply:
Fixed.